test/LibraryService.test.ts

```diff
@@ -0,0 +1,46 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
+import { serverConfig } from '../src/config/serverConfig';
+import { cleanUpDb, testSetup } from './utils';
+const supertest = require('supertest');
```

Comment:
Shouldn't need to use require here. If `import` gives you an error that says it can't find a declaration file, install they DefinitelyTyped types and it should be fine:


```
npm install --save-dev @types/supertest
```

The process should always be:

1. Ideally we don't get this error
2. If we do, hopefully there's a `@types/*` repo for it, try to install it
3. If there isn't one, hopefully it's simple enough to write our own `d.ts` file for it
4. If 1-3 fail, use require